### ISSUE TITLE
Overflow property create unnecessary vertical scroll on responsive tables

### ISSUE THREAD

**Issue by rhandrade** [twbs/bootstrap#16168]
In the Bootstrap 3.3.4, the `overflow` property in `table-responsive` class causes a bug when the table has a dropdown menu: A vertical scroll appears. (See image below)

![bug-bootstrap](https://cloud.githubusercontent.com/assets/4511733/6868814/77b949bc-d46d-11e4-94ab-e2b194e8d7af.png)

I created a jsFiddle to better demonstrate the problem. In this example show that removing the overflow property the problem does not happen. Link: http://jsfiddle.net/L4sjrska/9/

To reproduce the problem, pull the vertical slider as much as possible to the left. The result frame should be as large as possible.

Setting the `min-height` property to `100%` does not seem to be solving the problem. Putting a fixed size, as 300px, the problem back when the table is greater than the defined size.

Tested in Firefox 36.0.4, Chrome 41.0.2272.101 me Internet Explorer 11. All show the same behavior.

**Comment by twbs-lmvtfy** [comment]
Hi @rhandrade!

You appear to have posted a live example (http://fiddle.jshell.net/L4sjrska/8/show/light/), which is always a good first step. However, according to [the HTML5 validator](http://html5.validator.nu), **your example has some validation errors**, which might potentially be causing your issue:
* line 80, column 63: Duplicate ID ```` cargas ````.

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by rhandrade** [comment]
In the Bootstrap 3.3.4, the `overflow` property in `table-responsive` class causes a bug when the table has a dropdown menu: A vertical scroll appears. (See image below)

![bug-bootstrap](https://cloud.githubusercontent.com/assets/4511733/6868814/77b949bc-d46d-11e4-94ab-e2b194e8d7af.png)

I created a jsFiddle to better demonstrate the problem. In this example show that removing the overflow property the problem does not happen. Link: http://jsfiddle.net/L4sjrska/9/

To reproduce the problem, pull the vertical slider as much as possible to the left. The result frame should be as large as possible.

Setting the `min-height` property to `100%` does not seem to be solving the problem. Putting a fixed size, as 300px, the problem back when the table is greater than the defined size.

Tested in Firefox 36.0.4, Chrome 41.0.2272.101 me Internet Explorer 11. All show the same behavior.

**Comment by twbs-lmvtfy** [comment]
Hi @rhandrade!

You appear to have posted a live example (http://fiddle.jshell.net/L4sjrska/9/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 41, column 1: [E023](https://github.com/twbs/bootlint/wiki/E023): `.panel-body` must have a `.panel` or `.panel-collapse` parent

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by rhandrade** [comment]
In the Bootstrap 3.3.4, the `overflow` property in `table-responsive` class causes a bug when the table has a dropdown menu: A vertical scroll appears. (See image below)

![bug-bootstrap](https://cloud.githubusercontent.com/assets/4511733/6868814/77b949bc-d46d-11e4-94ab-e2b194e8d7af.png)

I created a jsFiddle to better demonstrate the problem. In this example show that removing the overflow property the problem does not happen. Link: http://jsfiddle.net/L4sjrska/11/

To reproduce the problem, pull the vertical slider as much as possible to the left. The result frame should be as large as possible.

Setting the `min-height` property to `100%` does not seem to be solving the problem. Putting a fixed size, as 300px, the problem back when the table is greater than the defined size.

Tested in Firefox 36.0.4, Chrome 41.0.2272.101 me Internet Explorer 11. All show the same behavior.

**Comment by patrickhlauke** [comment]
(for info, you can just comment "@twbs-lmvtfy retry" if you updated a fiddle, rather than reposting the entire comment again)

**Comment by patrickhlauke** [comment]
actually, was talking rubbish (got confused with twbs-savage here). shorter answer: if you simply post a short comment with the jsfiddle url, @twbs-lmvtfy will re-verify that fiddle :)

**Comment by rhandrade** [comment]
@patrickhlauke Sorry man. I'm I'm newbie here. :/ you want me to close this topic and open another, or delete my last comment and change the original post?

**Comment by patrickhlauke** [comment]
it's all good, no worries. just advice going forward :)

**Comment by cvrebert** [comment]
Duplicate of #16038.
